RecoTauTag/RecoTau/interface/RecoTauQualityCuts.h

```diff
@@ -43,7 +45,10 @@ namespace reco {
       explicit RecoTauQualityCuts(const edm::ParameterSet& qcuts);
 
       /// Update the primary vertex
-      void setPV(const reco::VertexRef& vtx) { pv_ = vtx; }
+      void setPV(const reco::VertexRef& vtx) {
+        pv_ = vtx;
+        pvPos_ = vtx->position();
```

Comment:
Hi Michal,

I have replaced pv_->position() by pvPos_ for two reasons: because pv_->position() was used in many places in the RecoTauQualityCuts.cc code and I wanted to be able to set the vertex position manually in RecoTauQualityCuts.h (implementing a function void RecoTauQualityCuts::setPVPos(const reco::Vertex::Point& vtxPos), which in the end I did not need). Is it ok to keep pvPos_ or should I switch back to using pv_->position() everywhere and remove the pvPos_ data-member ?

Reply:
I think it is better to not multiply beings, and not change unnecessarily code, so I vote to rollback to initial version.

Reply:
Ok, I have replaced pvPos_ by pv_->position() and removed the pvPos_ data-member. I also ran 'scram build code-format' again, so the formatting of the code should be the same as before.